subprojects/platform-base/src/integTest/groovy/org/gradle/language/base/LanguageTypeSampleIntegrationTest.groovy

```diff
@@ -16,17 +16,21 @@
 
 package org.gradle.language.base
 
-import org.gradle.integtests.fixtures.AbstractSampleIntegrationTest
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
 import org.gradle.integtests.fixtures.Sample
 import org.gradle.util.Requires
 import org.gradle.util.TestPrecondition
 import org.junit.Rule
 
 @Requires(TestPrecondition.ONLINE)
-class LanguageTypeSampleIntegrationTest extends AbstractSampleIntegrationTest {
+class LanguageTypeSampleIntegrationTest extends AbstractIntegrationSpec {
     @Rule
     Sample languageTypeSample = new Sample(temporaryFolder, "customModel/languageType")
 
+    def setup() {
+        executer.withGlobalRepositoryMirrors()
```

Comment:
`customModel/languageType` sample contains `buildSrc`.

Reply:
Please add the comment to the source code.

Reply:
Done.